scripts/extensions/sams/src/components/assets/assetGridItem.tsx

```diff
@@ -94,7 +95,8 @@ export class AssetGridItem extends React.PureComponent<IProps> {
             <GridItem
                 onClick={this.onItemClick}
                 onDoubleClick={this.onItemDoubleClick}
-                selected={this.props.selected || this.props.itemSelected}
+                singleClickSelected={this.props.selected || this.props.itemSelected}
+                doubleClickSelected={this.props.selected && this.props.itemSelectedForEdit}
```

Comment:
Again, using a utility function to set `locked` property here if the Asset is locked